### ISSUE TITLE
pbench-fio histogram log interval not settable

### ISSUE THREAD

**Issue by bengland2** [distributed-system-analysis/pbench#741]
The latency histogram logging interval is not settable and default is 60000 (msec), which means that for many pbench-fio tests no histogram log records will be written, which will be very confusing to most people.  Solution is to make this a settable parameter with default more like 10000 msec.

**Comment by bengland2** [comment]
I ran the new pbench-fio v3.2 RPM  and got some histogram logs, available [here](http://pbench.perf.lab.eng.bos.redhat.com/results/b06-h01-1029u/pbfio-iod32/fio__2018.01.10T18.41.52/1-randread-4KiB/sample5/clients/localhost/),  and when I run the histogram logging utility on them with the old default group_nr, 19, I get the error:

```
[root@b06-h01-1029u pbfio]# /usr/local/bin/fiologparser_hist.py -d 1000 -i 10000 --group_nr 19 fio_clat_hist.*log                                                            
Unable to determine bin values from input clat_hist files. Namely 
the first line of file 'fio_clat_hist.1.log' has 119 
columns of which we assume 116 correspond to histogram bins. 
This number needs to be equal to one of the following numbers:
```
However, when I don't specify --group_nr or when I set it to 29, which I believe is the new default, the script works fine:

```
[root@b06-h01-1029u pbfio]# /usr/local/bin/fiologparser_hist.py -d 1000 -i 10000 fio_clat_hist.*log                                                                          
end-time, samples, min, avg, median, 90%, 95%, 99%, max
10000, 9032562, 24, 140.698, 136.721, 177.876, 197.959, 245.578, 3702
20000, 17957270, 24, 142.394, 138.422, 178.684, 199.686, 246.868, 4259
30000, 18003255, 24, 140.295, 136.608, 176.120, 192.399, 245.968, 4259
40000, 18179662, 24, 139.718, 135.771, 176.814, 195.718, 245.728, 4259
50000, 18069681, 24, 141.691, 137.499, 178.940, 200.445, 246.785, 4259
60000, 8968566, 24, 141.467, 137.403, 178.789, 199.982, 246.168, 3702
```

So in conclusion, I think the new pbench-fio works but we should remove this from the fio job file
```
log_hist_coarseness=4 # 76 bins
```
And make sure that the histogram graph still works.  Also the histogram graph needs to have some units.  I'd suggest staying with microseconds by using the *-d 1000* option in the fiologparser_hist.py command, and putting it in the graph title.

**Comment by ndokos** [comment]
Assuming this refers to histogram_interval_msec, there is a command line option ---histogram_interval_msec=<int> that can be used to set it. But this is an obscure option whose meaning only experts know, so setting it to a reasonable default (10000) is important for the rest of us.  I also plan to move such defaults to the config file, so that people like you who are testing the nitty-gritty details of a benchmark can distribute experimental changes more easily, for testing.

**Comment by ndokos** [comment]
@bengland2 reminds me that yes indeed it is settable, because of *his* PR #746 :-)
